_plugins/page_to_api.rb

```diff
@@ -0,0 +1,16 @@
+# Copyright © Magento, Inc. All rights reserved.
```

Comment:
@dshevtsov  This came over from beta docs for P-rex, but should we remove to merge to master?